app/controllers/admin/creator_settings_controller.rb

```diff
@@ -0,0 +1,39 @@
+module Admin
+  class CreatorSettingsController < Admin::ApplicationController
+    before_action :extra_authorization, only: [:create]
+
+    # def success?
+    #   errors.none?
+    # end
+
+    def new; end
+
+    def create
+      # errors = []
+      # binding.pry
+      ::Settings::Community.community_name = settings_params[:community_name]
+      ::Settings::General.logo_svg = settings_params[:logo_svg]
+      ::Settings::UserExperience.primary_brand_color_hex = settings_params[:primary_brand_color_hex]
+      ::Settings::Authentication.invite_only_mode = settings_params[:invite_only]
+      ::Settings::UserExperience.public = settings_params[:public]
+
+      # rescue ActiveRecord::RecordInvalid => e
+      #   errors << e.message
+    end
+
+    private
+
+    def extra_authorization
+      not_authorized unless current_user.has_role?(:super_admin)
+    end
+
+    def settings_params
+      allowed_params = { community_name: ::Settings::Community.community_name,
```

Comment:
Why is this a hash with values if we only need the keys? Wouldn't an array be better here? Or will you eventually use the values?